docs/release-notes/8.10.asciidoc

```diff
@@ -0,0 +1,43 @@
+[[release-notes-header-8.10.0]]
+== 8.10
+
+[discrete]
+[[release-notes-8.10.0]]
+=== 8.10.0
+
+[discrete]
+[[breaking-changes-8.10.0]]
+==== Breaking changes
+There are no breaking changes in 8.10.0.
+
+[discrete]
+[[features-8.10.0]]
+==== New features
+* Adds the MITRE ATT&CK® coverage page, which shows how well your active detection rules protect against adversary tactics and techniques ({pull}161556[#161556], {pull}163498[#163498], {pull}164613[#164613], {pull}164986[#164986]).
+* Adds a component under the Elastic AI Assistant header that allows you to select a different connector ({pull}163666[#163666]).
+* Adds a flyout that allows you to examine rule details when installing or updating a prebuilt rule ({pull}163304[#163304]).
+* Adds the ability to specify custom highlighted fields for an alert ({pull}163235[#163235]).
+* Adds role-based access control for the Elastic AI Assistant ({pull}163031[#163031]).
```

Comment:
This isn't mandatory, but might make logical sense to group similar features together. So in this case, I'd put this RBAC for AI Assistant beneath the other AI feature on line 17. Same idea with rules, alerts, etc. Just a thought!

Reply:
Thanks, @jmikell821! Resolved in [2d9a7c0](https://github.com/elastic/security-docs/pull/3781/commits/2d9a7c078836966792b465741d0d0352a02f074b).